### PR TITLE
fix: Teaching bubble carousel tabs are visible in windows high contrast

### PR DESCRIPTION
**Pull request by smhigley**
## Previous Behavior

The tabs/dots were not visible in WHCM:
screenshot of the teaching popover in dark and light high contrast themes, showing no visible dots between the previous and next buttons

## New Behavior

screenshot of the teaching popover in dark high contrast, showing visible dots

## Related Issue(s)

- Fixes an ADO a11y issue

### CODE REVIEW

**Comment by fabricteam** [comment]
## [Perf Analysis (`@fluentui/react-components`)](https://github.com/microsoft/fluentui/wiki/Perf-Testing)
<p>No significant results to display.</p><details><summary>All results</summary><p>
  <table>
  <tr>
    <th>Scenario</th>
    <th>Render type</th>
    <th>
      <a href="https://github.com/microsoft/fluentui/wiki/Perf-Testing#why-are-results-listed-in-ticks-instead-of-time-units">Master Ticks</a>
    </th>
    <th>
      <a href="https://github.com/microsoft/fluentui/wiki/Perf-Testing#why-are-results-listed-in-ticks-instead-of-time-units">PR Ticks</a>
    </th>
    <th>Iterations</th>
    <th>Status</th>
  </tr><tr>
            <td>Avatar</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/31734/perf-test-react-components/Avatar-mount_base.html">647</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/31734/perf-test-react-components/Avatar-mount.html">622</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Button</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/31734/perf-test-react-components/Button-mount_base.html">304</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/31734/perf-test-react-components/Button-mount.html">298</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Field</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/31734/perf-test-react-components/Field-mount_base.html">1128</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/31734/perf-test-react-components/Field-mount.html">1136</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>FluentProvider</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/31734/perf-test-react-components/FluentProvider-mount_base.html">722</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/31734/perf-test-react-components/FluentProvider-mount.html">720</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>FluentProviderWithTheme</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/31734/perf-test-react-components/FluentProviderWithTheme-mount_base.html">82</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/31734/perf-test-react-components/FluentProviderWithTheme-mount.html">84</a></td>
            <td>10</td>
            <td></td>
           </tr>
<tr>
            <td>FluentProviderWithTheme</td>
            <td>virtual-rerender</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/31734/perf-test-react-components/FluentProviderWithTheme-virtual-rerender_base.html">30</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/31734/perf-test-react-components/FluentProviderWithTheme-virtual-rerender.html">35</a></td>
            <td>10</td>
            <td></td>
           </tr>
<tr>
            <td>FluentProviderWithTheme</td>
            <td>virtual-rerender-with-unmount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/31734/perf-test-react-components/FluentProviderWithTheme-virtual-rerender-with-unmount_base.html">85</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/31734/perf-test-react-components/FluentProviderWithTheme-virtual-rerender-with-unmount.html">86</a></td>
            <td>10</td>
            <td></td>
           </tr>
<tr>
            <td>MakeStyles</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/31734/perf-test-react-components/MakeStyles-mount_base.html">858</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/31734/perf-test-react-components/MakeStyles-mount.html">855</a></td>
            <td>50000</td>
            <td></td>
           </tr>
<tr>
            <td>Persona</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/31734/perf-test-react-components/Persona-mount_base.html">1780</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/31734/perf-test-react-components/Persona-mount.html">1754</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>SpinButton</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/31734/perf-test-react-components/SpinButton-mount_base.html">1410</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/31734/perf-test-react-components/SpinButton-mount.html">1402</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>SwatchPicker</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/31734/perf-test-react-components/SwatchPicker-mount_base.html">1630</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/31734/perf-test-react-components/SwatchPicker-mount.html">1672</a></td>
            <td>5000</td>
            <td></td>
           </tr></table></p></details>

<div id="perfComment9425"/>

**Comment by fabricteam** [comment]
## 📊 Bundle size report

| Package & Exports | Baseline (minified/GZIP) | PR    | Change     |
| :---------------- | -----------------------: | ----: | ---------: |
| <samp>react-components</samp> <br /> <abbr title='bundle-size/ReactComponents.fixture.js'>react-components: entire library</abbr> | `1.093 MB`<br />`270.378 kB` | `1.093 MB`<br />`270.372 kB` | `155 B` <br />`-6 B` |

<details>
<summary>Unchanged fixtures</summary>

| Package & Exports | Size (minified/GZIP) |
| ----------------- | -------------------: |
| <samp>react-components</samp> <br /> <abbr title='bundle-size/ButtonProviderAndTheme.fixture.js'>react-components: Button, FluentProvider & webLightTheme</abbr> | `69.141 kB`<br />`20.157 kB` |
| <samp>react-components</samp> <br /> <abbr title='bundle-size/MultipleComponents.fixture.js'>react-components: Accordion, Button, FluentProvider, Image, Menu, Popover</abbr> | `211.741 kB`<br />`60.957 kB` |
| <samp>react-components</samp> <br /> <abbr title='bundle-size/ProviderAndTheme.fixture.js'>react-components: FluentProvider & webLightTheme</abbr> | `44.442 kB`<br />`14.607 kB` |
| <samp>react-portal-compat</samp> <br /> <abbr title='bundle-size/PortalCompat.fixture.js'>PortalCompatProvider</abbr> | `8.39 kB`<br />`2.64 kB` |
| <samp>react-timepicker-compat</samp> <br /> <abbr title='bundle-size/TimePicker.fixture.js'>TimePicker</abbr> | `103.427 kB`<br />`34.598 kB` |
</details>
<sub>🤖 This report was generated against <a href='https://github.com/microsoft/fluentui/commit/5f1feb5721933fb703d789fb00ce385d7e3f950e'>5f1feb5721933fb703d789fb00ce385d7e3f950e</a></sub>
<div id="bundleSizeComment9423"/>

**Review by Mitch-At-Work — APPROVED** [review]

File: change/@fluentui-react-teaching-popover-cdaeeec1-2b74-432f-a6ec-694d78207d7c.json [hunk 1]
```diff
@@ -0,0 +1,7 @@
+{
```
Comment (fabricteam) [inline change/@fluentui-react-teaching-popover-cdaeeec1-2b74-432f-a6ec-694d78207d7c.json:1]: ## [🕵 fluentuiv9 Open the Visual Regressions report to inspect the affected screenshots]( https://staticpagerelease.z16.web.core.windows.net/azurestaticapps/vr-approval/index.html?organization=uifabric&project=fabricpublic&prId=31734&commitId=c79607c57e5426da364eaa25f6673137468f677b&env=prod&clientType=FLUENTUI)


  <details><summary><b>Avatar Converged</b> 3 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Avatar Converged.badgeMask - RTL.normal.chromium.png|3| Changed|
|Avatar Converged.basic - Dark Mode.normal.chromium.png|0| Removed|
|Avatar Converged.basic - RTL.normal.chromium.png|0| Removed|

</details>




<div id="vrtCommentfluent-ui_VRT_Pipeline_v9"/>

Comment (smhigley) [reply change/@fluentui-react-teaching-popover-cdaeeec1-2b74-432f-a6ec-694d78207d7c.json:1]: unrelated to this PR

**Pull request merged**